### PR TITLE
NODE-5501 Fix broken build on MacOS aarch64 architecture

### PR DESCRIPTION
**Pull request by alex-drongo**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -42,13 +42,26 @@ PYTEST_WORKERS        ?= 10
 PYTEST_ARGS           ?= --allure-features=Node
 TEST_RS               ?= false
 
+WALLARM_LABELS		  ?='group=default_docker_smoke_test'
```
Comment (xDmitriev) [inline Makefile]:
```suggestion
WALLARM_LABELS		  ?='group=defaultDockerNode'
```

File: .env.example [hunk 2]
```diff
@@ -7,7 +8,13 @@ WALLARM_API_CA_VERIFY=True
...
-# Pytest variables (optional)
-# PYTEST_WORKERS=10
-# PYTEST_ARGS="--allure-features=Node"
+PYTEST_WORKERS=10
+PYTEST_ARGS="--allure-features=Node"
+ALLURE_ENVIRONMENT_ARCH = ...
+NODE_VERSION=...
+WEBHOOK_API_KEY=...
+WEBHOOK_UUID=...
+# not used when started localy
```
Comment (xDmitriev) [inline .env.example]: if it's not required for a local run we do not need it here, cuz `.env` file is for local runs only

File: .env.example [hunk 3]
```diff
@@ -1,4 +1,5 @@
 # Wallarm API settings
+NODE_IMAGE = "wallarm/node:test"
```
Comment (xDmitriev) [inline .env.example]: is this variable required for local build and test? 

`NODE_MAGE` overwrites here by `$(IMAGE)`
https://github.com/wallarm/docker-wallarm-node/pull/101/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R25

File: .env.example (hunk 2 above)
Comment (alexey-ponomarev-clearscale) [reply .env.example]: updated

File: .env.example (hunk 3 above)
Comment (alexey-ponomarev-clearscale) [reply .env.example]: no, that parameter is not required for local start.

**Pull request closed without merging**